spec/controllers/slides_controller_spec.rb

```diff
@@ -47,6 +47,19 @@
       expect(response.status).to eq(200)
       expect(response.body).to include(I18n.t(:popular_slides))
     end
+
+    it 'returns collect header' do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response.body).to include("navbar-inverse")
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->